.github/workflows/ci.yml

```diff
@@ -49,6 +49,50 @@ jobs:
       codeql: ${{ github.event_name == 'schedule' }}
       upload_artifacts: ${{ matrix.os == 2022 }}
 
+  build_allpackage:
+    name: Build AllPackage
+    needs: build
+    strategy:
+      fail-fast: false
+      matrix:
+        config: [Release, Debug]
+    permissions:
+      actions: read
+      contents: read
+    runs-on: windows-2022
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@d632683dd7b4114ad314bca15554477dd762a938
+      with:
+        repository: microsoft/xdp-for-windows
+        submodules: recursive
+        ref: ${{ inputs.ref }}
```

Comment:
Not really necessary unless you're exposing a template for others to call.

Reply:
Ah, this was probably copied from the build.yml template.